packages/terra-application-links/src/tabs/_TabUtils.js

```diff
@@ -0,0 +1,12 @@
+const KEYCODES = {
```

Comment:
It doesn't appears that space, left_arrow and right_arrow keys are used

Reply:
Removed unneeded values.